llvm/docs/AMDGPUUsage.rst

```diff
@@ -583,8 +583,8 @@ Generic processor code objects are versioned. See :ref:`amdgpu-generic-processor
                                                                                                   - ``v_dot2_f32_f16``
 
 
-     ``gfx9-4-generic``   ``amdgcn``     - ``gfx940``      - xnack            - Absolute flat   FP8 and BF8 instructions,
-                                         - ``gfx941``      - sramecc            scratch         FP8 and BF8 conversion instructions,
+     ``gfx9-4-generic``   ``amdgcn``     - ``gfx940``      - xnack            - Architected     FP8 and BF8 instructions,
+                                         - ``gfx941``      - sramecc            flat scratch    FP8 and BF8 conversion instructions,
```

Comment:
how about `Packed work-item IDs`? I think I copied the wrong stuff here Lol.

Reply:
it seems like `tgsplit` and `kernarg preload` are also supported.

Reply:
Yes, gfx94x and gfx950 have packed work-item IDs, so I think the gfx9-4-generic processor requires them as well.

gfx94x and gfx950 offer both tgsplit and kernarg preload. so gfx9-4-generic can support the feature. These are optional, so gfx9-4-generic is not necessarily _required_ to offer them. But it probably should and they probably should be in this table.

Reply:
Fixed.